.github/workflows/build.yml

```diff
@@ -1,26 +1,101 @@
 name: Build oee-simulators
 
 on:
-  workflow_call:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: Name of the environment to use
+        default: "oee-dev"
+        options:
+          oee-dev: oee-dev Environment
+        required: true
+
+      release_body:
+        type: string
+        description: GitHub Release Message
+        default: "Release version ${GITHUB_RUN_NUMBER}"
+
+      run_release:
+        description: 'Run release workflow?'
+        required: false
+        default: false
+        type: bool
+
+      run_build:
```

Comment:
is that input really required/used?

Reply:
- I'd think that we call release.yml if we want to create a release; this will then call build.yml to create the binary
- I'd call release.yml directly if I want to create a release